.github/workflows/build_test.yml

```diff
@@ -44,6 +43,12 @@ jobs:
           git config --global user.name "Git User"
           west patch
 
+          pip3 install -r zephyr/scripts/requirements-base.txt
+          pip3 install -r zephyr/scripts/requirements-build-test.txt
+          pip3 install -r zephyr/scripts/requirements-run-test.txt
+          # Needed for TF-M
+          pip3 install cryptography pyasn1 pyyaml cbor>=1.0.0 imgtool>=1.9.0 jinja2 click
```

Comment:
Can we make this list in alphabetical order? I'm okay keeping it as is, if there is a reason for it (like being copy-pasted from other place).